doc/cortx-minikube-k8s-installation.md

```diff
@@ -0,0 +1,258 @@
+# **CORTX on MINIKUBE - Quick Install Guide**
+
+**1. Minimum Requirements:**
+
+* **RAM**: 10GB
+* **Processor**: 6
+* **NIC**: 1
+* **OS Disk**: 1 disk of 20GB
+* **Data Disks**: 4 disks of 10GB each
+* **Metadata disks**: 2 disks of 10GB each
+* **Container or virtual machine manager**, such as: Docker, Hyperkit, Hyper-V, KVM, Parallels, Podman, VirtualBox, or VMware Fusion/Workstation
+
+
+**2. Install Kubectl**
+
+
+```
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+```
+ 
+```
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+
+```
+kubectl version
+```
+
+**3. Install Helm(A package manager for K8s):**
+
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+```
+
+```
+chmod 700 get_helm.sh
+```
+
+```
+./get_helm.sh
+```
+
+**4. Install and start minikube(Centos7.9):**
+
+```
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+```
+
+```
+sudo install minikube-linux-amd64 /usr/local/bin/minikube
+```
+
+```
+minikube start --driver=none
+```
+
+Note minikube may fail to start in some setup due to driver issue, if this happens you can install the drivers here -> Drivers | minikube (k8s.io)
+
+
+**5. Clone Cortx-K8s framework**
+
+```
+git clone https://github.com/Seagate/cortx-k8s
+```
+
+```
+cd cortx-k8s/k8_cortx_cloud/
+```
+
+**5.1  Update the solution.yaml file.**
+
+- Update the node name in `solution.yaml` with the name of the node you get from `kubectl get node` command. 
+
+    ```
+    nodes:
+        node1:
+          name: control-plane.minikube.internal
+
+    ```
+
+- Locate and change the setup_size to small:
+    ```
+    setup_size: small
+    ```
+
+- Update the device name and size for all the cvg(s). For e.g, We have added `/dev/sdb`, `/dev/sdc`, `/dev/sde`, `/dev/sdf` for the disk and `10Gi` for the size in the below snip.
```

Comment:
@alfhad Is it possible to include the command to check the storage devices; like `lsblk`